integration_tests/e2e/bookingJourney.cy.ts

```diff
@@ -142,18 +143,24 @@ context.skip('Booking journey', () => {
     // Main contact
     const mainContactPage = Page.verifyOnPage(MainContactPage)
     mainContactPage.selectVisitorByName('Adult One')
-    mainContactPage.checkHasPhoneNumber()
-    mainContactPage.enterPhoneNumber('01234 567 890')
+    mainContactPage.continue()
+
+    // Contact details
+    const contactDetailsPage = Page.verifyOnPage(ContactDetailsPage, 'Adult One')
+    contactDetailsPage.checkGetUpdatesByEmail()
+    contactDetailsPage.enterEmail('adult.one@example.com')
+    contactDetailsPage.checkGetUpdatesByPhone()
+    contactDetailsPage.enterPhoneNumber('07712 000 000')
     cy.task('stubChangeVisitApplication', {
       ...application,
-      visitContact: { name: 'Adult One', telephone: '01234 567 890' },
+      visitContact: { name: 'Adult One', telephone: '07712 000 000', email: 'adult.one@example.com' },
```

Comment:
Updating the dummy number used in various test files to be a mobile one, so it enables, e.g. messages about texts being sent that are now only shown if the number type is mobile.

(Originally tried to use a number in Ofcom's `07700 900000 to 900999` 'dummy' unallocated range but these failed `libphonenumber-js`'s validation. ChatGTP suggested `07712 000 000` - which it says has a valid mobile prefix but invalid subscriber portion 😅 !)

Reply:
yeah, tried those ones (they're on the Ofcom site - I found them linked to from the GOVUK Notify API docs) but they failed **because** they're unallocated I think. This one that ChatGTP suggested seems to work because it's a valid operator prefix but apparently the `000000` bit would never be allocated 🤷‍♂️